### PR TITLE
Provide a management command to generate download URLs for message store data.

### PR DESCRIPTION
**Pull request by smn**
Sometimes when doing a bulk download, piecing together the URLs one by one is super tedious.

### CODE REVIEW

**Comment by smn** [comment]
ready for review

File: go/base/tests/test_go_generate_export_conversations_urls.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    base_url = 'http://localhost:1234/export/'
+
+    def setUp(self):
+        self.vumi_helper = self.add_helper(DjangoVumiApiHelper())
+        self.user_helper = self.vumi_helper.make_django_user()
+        self.user_email = self.user_helper.get_django_user().email
+
+        self.command = go_generate_export_conversations_urls.Command()
+        self.command.stdout = StringIO()
+        self.command.stderr = StringIO()
```
Comment (hodgestar) [inline go/base/tests/test_go_generate_export_conversations_urls.py]: I recently discovered that a better pattern is to use `call_command`. See e.g. https://github.com/praekelt/vumi-go/blob/883d8d6f62101cee137e12f1579a32bc2a820ee7/go/base/tests/test_go_system_stats.py#L25-L29

**Comment by hodgestar** [comment]
Other than the small suggestion for the tests, looks good.

**Comment by smn** [comment]
@hodgestar thanks, ready for re-review.

**Comment by hodgestar** [comment]
:+1:

**Comment by hodgestar** [comment]
(although we should probably give Travis a chance to build before landing)

**Pull request merged**